frontend/javascripts/viewer/model/actions/ui_actions.ts

```diff
@@ -201,10 +201,14 @@ export const hideMeasurementTooltipAction = () =>
   ({
     type: "HIDE_MEASUREMENT_TOOLTIP",
   }) as const;
-export const setLastMeasuredPositionAction = (position: Vector3) =>
+export const setLastMeasuredAndViewportPositionAction = (
+  lastMeasuredPosition: Vector3,
+  viewportPosition: Point2,
```

Comment:
I optimized displaying the distance / area measurement tool tip a little. Previously `lastMeasuredPosition` was transformed back into the space of the plane to get the location where to show the tooltip. Now this position is directly stored in the store to avoid the back and forth calculation. This makes the code in the tooltip component also easier